src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao dao;
+
+    public ActorServiceImpl(ActorDao dao) {
+        this.dao = dao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return dao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        Optional<Actor> result = dao.get(id);
+        return result.orElseGet(Actor::new);
```

Comment:
very  confusing logic... if our entity does not exists, your code would create new. but it's not what we asked for by id. consider using `orElseThrow()`